src/components/Notifications/types.ts

```diff
@@ -365,6 +365,35 @@ type StreamingNotificationsProps =
   | StreamingTimeExceededNotificationProps
   | StreamingPlaceUpdatedNotificationProps
 
+// Referral Notifications
+
+type CommonReferralNotificationMetadataProps = {
+  title: string
+  description: string
+  address: string
+  tier: number
+  url: string
+  image: string
+  invitedUserAddress: string
+  invitedUsers: number
+  rarity?: Rarity
+}
```

Comment:
Is rarity always included in the `ReferralNewTierReachedNotificationProps`? If that's so, what do you think about doing something like this:
```suggestion
type ReferralNotificationMetadataProps = {
  title: string
  description: string
  address: string
  tier: number
  url: string
  image: string
  invitedUserAddress: string
  invitedUsers: number
}

type NewTierReachedNotificationMetadata = ReferralNotificationMetadataProps & { rarity: Rarity }
```